.buildkite/pipeline.yml

```diff
@@ -79,7 +79,7 @@ steps:
           context: "UI Tests (iPhone)"
 
   - label: "🔬 UI Tests (iPad)"
-    command: .buildkite/commands/run-ui-tests.sh UITests "iPad Air (4th generation)" 15.0
+    command: .buildkite/commands/run-ui-tests.sh UITests "iPad Air (5th generation)" 15.0
```

Comment:
Only one non-blocking comment for this change. As it was discussed [in this PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/19193#discussion_r953666294) (for WPiOS), specifying the iOS version does not really affect the used iOS version. Xcode will still use the only one it has available ([which is 15.5](https://buildkite.com/automattic/woocommerce-ios/builds/7712#0183102a-7bbd-4588-a32f-17c4bd455cba/788-858) for Xcode 13.4.1). However, passing the 15.0 parameter is reflected in the execution log, so it might give a false impression that 15.0 was really used.

Reply:
I didn't notice this issue, thanks for pointing this out. I'll follow up in a separate PR 👍

Reply:
Follow-up PR: https://github.com/woocommerce/woocommerce-ios/pull/7672